core/src/main/scripts/importer/cbioportalImporter.py

```diff
@@ -14,7 +14,7 @@
 from pathlib import Path
 
 # configure relative imports if running as a script; see PEP 366
-if __name__ == "__main__" and __package__ is None:
+if __name__ == "__main__" and (__package__ is None or __package__ is ''):
```

Comment:
As I responded above, this undocumented workaround does not belong in the code.

Reply:
Change was reverted.